libsolidity/interface/CompilerStack.h

```diff
@@ -251,6 +251,10 @@ class CompilerStack: boost::noncopyable
 	/// @returns the parsed source unit with the supplied name.
 	SourceUnit const& ast(std::string const& _sourceName) const;
 
+	/// @returns the parsed contract with the supplied name. Throws an exception if the contract
+	/// does not exist.
+	ContractDefinition const& contractDefinition(std::string const& _contractName) const;
+
```

Comment:
Is there any particular reason this was private? I made it public so that I can access the contract and its annotations without having to create special methods for that.